ocos/kernels/string_split.cc

```diff
@@ -13,9 +13,9 @@ void KernelStringSplit::Compute(OrtKernelContext* context) {
   const OrtValue* input_sep = ort_.KernelContext_GetInput(context, 1);
   const OrtValue* input_skip_empty = ort_.KernelContext_GetInput(context, 2);
   const bool* skip_empty = ort_.GetTensorData<bool>(input_skip_empty);
-  std::vector<std::string> X, sep;
-  GetTensorMutableDataString(api_, ort_, context, input_X, X);
-  GetTensorMutableDataString(api_, ort_, context, input_sep, sep);
+  std::vector<std::wstring> X, sep;
+  GetTensorMutableDataWString(api_, ort_, context, input_X, X);
+  GetTensorMutableDataWString(api_, ort_, context, input_sep, sep);

```

Comment:
Does it mean users should input std::wstring instead of std::string?

Reply:
Not necessarily but it is safer to do it. It matters to split strings, it does not for join. Concatenating string or wstring is the same.